daffodil-lib/src/test/scala/LineCounter.scala

```diff
@@ -30,33 +30,48 @@ object LineCounter extends App {
   /**
    * What file extensions to count.
    */
-  val fileSuffixesToInclude = List(".scala", ".java", ".tdml", ".xsd") // not .xml as some of those are data files that are giant.
+  val fileSuffixesToInclude = List(".scala", ".java", ".tdml", ".xsd", ".tdml.xml", ".xsl", ".sch", ".sbt") // not just .xml as some of those are data files that are giant.
+
+  /**
+   * What to exclude (subversion artifacts, anything generated, version control stuff, caches, settings, etc.)
+   */
+  val directoriesToExclude = List(".svn", ".git", ".settings", "lib_managed", "src_managed", "resource_managed", "bin", ".cache", "target", "lib", ".idea")
 
   /**
    * Where is this freakin' sandbox anyway....edit to point to yours.
    */
   val root = "/home/mbeckerle/dataiti/git/"
 
+  val daffodilDirsCountedSeparately = List("daffodil-test",
+    "daffodil-test-ibm1",
+    "tutorials",
+    "test-stdLayout")
   /**
-   * What modules within the sandbox to count
+   * What modules within the daffodil subtree sandbox to count
    */
-  val srcModulesToInclude = List("daffodil-core", "daffodil-runtime1", "daffodil-runtime1-unparser", "daffodil-lib", "daffodil-propgen", "daffodil-tdml", "daffodil-io", "daffodil-cli", "daffodil-japi", "daffodil-sapi")
-  val unitTestModulesToInclude = srcModulesToInclude
+  val srcModulesToInclude = {
+    val daf = new File(root + "daffodil")
```

Comment:
This assumes the daffodil source directory is called "daffodil". Which isn't always true. Some other common names are going to be "incubator-daffodil" or "daffodil.git", "incubator-daffodil.git", etc. This shouldn't be hardcodede in here.